packages/shylu/shylu_node/basker/src/shylubasker_nfactor_diag.hpp

```diff
@@ -395,8 +400,8 @@ namespace BaskerNS
 
     //#define BASKER_DEBUG_NFACTOR_DIAG
     #ifdef BASKER_DEBUG_NFACTOR_DIAG
-    if (Options.verbose == BASKER_TRUE) {
-      //printf( " c: %d bcol=%d, brow2=%d, wsize=%d\n", (int)c, (int)bcol, (int)brow2, (int)ws_size );
+    if (Options.verbose == BASKER_TRUE && c == 4) {
```

Comment:
Note for later cleanup (future PR if preferred), in case c is tied to particular matrix